src/accounts.cs

```diff
@@ -127,29 +133,40 @@ interface ACAccountType : NSSecureCoding {
 		[Export ("accessGranted")]
 		bool AccessGranted { get;  }
 
+		[Deprecated (PlatformName.iOS, 11, 0, message: "Use Twitter SDK instead")]
```

Comment:
https://github.com/xamarin/xamarin-macios/wiki/BINDINGS#availability-attributes-messages (:

Please add a dot at the end of the sentance.